crates/analyzer/src/handlers/create_symbol_table.rs

```diff
@@ -448,19 +450,33 @@ impl<'a> VerylGrammarTrait for CreateSymbolTable<'a> {
                 .push(arg.identifier.identifier_token.token.text);
 
             for item in items {
-                let direction: crate::symbol::Direction = item.direction.as_ref().into();
-                let property = ModportMemberProperty { direction };
-                let id = self.insert_symbol(
-                    &item.identifier.identifier_token.token,
-                    SymbolKind::ModportMember(property),
-                    false,
-                );
-                members.push(id);
+                let kind = match &*item.direction {
+                    Direction::Ref(_) | Direction::Modport(_) => {
+                        continue;
+                    }
+                    Direction::Import(_) => {
+                        let property = ModportFunctionMemberProperty {
+                            function: SymbolId::default(),
```

Comment:
Why the appropriate id can't be determined at this point from `self.function_ids`?

Reply:
This is to support function definitions located after a modoport declaration. Variable declarations can be after a modport declarations so I thought function definitions should be.

Reply:
OK.